src/main/java/org/openjdk/jextract/impl/OutputFactory.java

```diff
@@ -64,11 +64,7 @@ JavaSourceFile[] generate(Declaration.Scoped decl) {
     }
 
     private void generateDecl(Declaration tree) {
-        try {
-            tree.accept(this, null);
-        } catch (Exception ex) {
-            ex.printStackTrace();
-        }
+        tree.accept(this, null);
```

Comment:
the try/catch seemed suspicious, and was also printing on std err, so I dropped that (after all, if some unexpected exception happens, we want to know?)

Reply:
I suppose this was added when jextract was less stable, or lacked filtering support. So, if we got an exception while generating one of the trees, jextract would still output _something_